docs/sources/source/index.rst

```diff
@@ -23,7 +23,7 @@ A "Neural Module" is a block of code that computes a set of outputs from a set o
 
 Neural Modules’ inputs and outputs have Neural Type for semantic checking.
 
-An application built with NeMo application is a Directed Acyclic Graph (DAG) of connected modules enabling researchers to define and build new speech and nlp networks easily through API Compatible modules.
+An application built with a NeMo application is a Directed Acyclic Graph (DAG) of connected modules enabling researchers to define and build new speech and nlp networks easily through API Compatible modules.
```

Comment:
`An application built with NeMo is a ...`

Reply:
Correcting.